### PR TITLE
deduplicating renames

### PR DESCRIPTION
**Pull request by jrieken**
fix for #175

### CODE REVIEW

File: src/OmniSharp/Models/LinePositionSpanTextChange.cs [hunk 1]
```diff
@@ -61,5 +61,26 @@ public class LinePositionSpanTextChange
...
+            return NewText == other.NewText
+                && StartLine == other.StartLine
+                && StartColumn == other.StartColumn
+                && EndLine == other.EndLine
+                && EndColumn == other.EndColumn;
+        }
+        
+        public override int GetHashCode()
+        {
+            return NewText.GetHashCode() + 23 * StartLine + 23 * StartColumn + 23 * EndLine + 23 * EndColumn;
```
Comment (nosami) [inline src/OmniSharp/Models/LinePositionSpanTextChange.cs]: This isn't a good hashcode function. Easy to create dupes.

Comment (nosami) [inline src/OmniSharp/Models/LinePositionSpanTextChange.cs]: Line 1, Column 10 or Line 10, Column 1 ?

**Comment by jrieken** [comment]
@nosami - pushed a better hash function. btw - is there a way to tell travis not to use mono 4.0.1

**Comment by nosami** [comment]
Looks much better. I'll test it out later.

You can do pretty much anything you want on Travis. See here https://github.com/OmniSharp/omnisharp-server/blob/master/.travis.yml#L1-L7. It's even possible to run build against a matrix of mono versions. 
The only restriction that I'm aware of is that you can't use Windows ;)

**Comment by nosami** [comment]
Looks good!

**Pull request merged**